src/plots/Volume/avtVolumeRenderer.C

```diff
@@ -7,26 +7,48 @@
 // ************************************************************************* //
 
 #include "avtVolumeRenderer.h"
+
 #include <visit-config.h>
-#include <vtkDataSet.h>
+
+#include <StackTimer.h>
+#include <DebugStream.h>
+#include <ImproperUseException.h>
+#include <InvalidVariableException.h>
+#include <avtCallback.h>
+
+#include <vtkColorTransferFunction.h>
+#include <vtkImageData.h>
+#include <vtkPiecewiseFunction.h>
+#include <vtkCellData.h>
 #include <vtkPointData.h>
 #include <vtkRectilinearGrid.h>
-#include <vtkUnstructuredGrid.h>
 #include <vtkRenderer.h>
-#include <vtkRenderWindow.h>
-#include <vtkToolkits.h>
-#include <math.h>
-#include <float.h>
+#include <vtkGPUVolumeRayCastMapper.h>
+#include <vtkSmartVolumeMapper.h>
+#include <vtkVolume.h>
+#include <vtkVolumeProperty.h>
 
-#include <avtCallback.h>
-#include <avtVolumeRendererImplementation.h>
-#include <avtDefaultRenderer.h>
+#ifdef HAVE_OSPRAY
+    #include <vtkOSPRayVolumeMapper.h>
+    #include <vtkOSPRayRendererNode.h>
+#endif
 
-#include <DebugStream.h>
-#include <ImproperUseException.h>
-#include <InvalidLimitsException.h>
-#include <StackTimer.h>
-#include <VolumeFunctions.h>
+#include <vtkGeometryFilter.h>
+#include <vtkPolyDataMapper.h>
+
+#ifndef NO_DATA_VALUE
```

Comment:
This copies a definition elsewhere and so am assuming both definitions need to be kept consistent. Just wondering if there is a better *single* place to define it for both so there is no chance of them getting out of sync. And, again IMHO the symbol name doesn't really match its use.